--- mltex.ch
+++ mltex.ch
@@ -284,26 +284,15 @@
 @z
 
 @x [1] m.2 l.187 - MLTeX: change banner line
-@d banner=='This is TeX, Version 3.14159' {printed when \TeX\ starts}
+November 1984].
 @y
+November 1984].
 
 ML\TeX{} will add new primitives changing the behaviour of \TeX.  The
 |banner| string has to be changed.  We do not change the |banner|
 string, but will output an additional line to make clear that this is
 a modified \TeX{} version.
 
-If you plan to add a command line option ``mltex'' to your \TeX{}
-implementation, use the boolean variable |mltex_p| to reflect this
-option.  The boolean |mltex_p| should be set {\it before\/} or in the
-initialisation routine |initialize| in the Ini\TeX{} run.  (You can
-not use a ``mltex'' option in Vir\TeX{}.)  To add this option, remove
-the definition of |mltex_p| below and add some code to set it.
-
-The boolean value of |mltex_p| is dumped to the \.{FMT} file and
-undumped from this file to the boolean |mltex_enabled_p|.
-
-@d mltex_p==true            {enable ML\TeX{} primitives}
-@d banner=='This is TeX, Version 3.14159' {printed when \TeX\ starts}
 @z
 
 
